wiz-admission-controller/Chart.yaml

```diff
@@ -7,7 +7,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 3.6.0
```

Comment:
why minor here?
Is this a breaking change? it's only a chart change, not image, so IMHO you can only bump patch

Reply:
Done